### PR TITLE
Feature/compatible 2.0

### PR DESCRIPTION
**Pull request by ultraheroe**
This will add compatibility (I hope 😄 ) with the new Apache Airflow 2.0 version

### CODE REVIEW

**Review by vicsufer — APPROVED** [review]

**Review by caldito — APPROVED** [review]

**Review by jmonterrubio — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@ tests/__pycache__
 .cache
 .vagrant
```
Comment (dortegau) [inline .gitignore:6]: As this role is not tested using vagrant anymore, I think we could remove `.vagrant` entry

**Review by frantsao — APPROVED** [review]

**Comment by blalop** [comment]
Could you please add a [.gitattributes file](https://github.com/idealista/cookiecutter-ansible-role/blob/develop/%7B%7Bcookiecutter.app_name%7D%7D_role/%7B%25%20raw%20%25%7D.gitattributes%7B%25%20endraw%20%25%7D)?

**Review by jperera — APPROVED** [review]

**Review by emepege — APPROVED** [review]

**Review by sorobon — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,112 +1,142 @@
...
-![Ansible](https://img.shields.io/badge/ansible-2.4.5.0-green.svg)
+![Ansible](https://img.shields.io/badge/ansible-2.9.9-green.svg)
 
-## Versioning
+## Versioning 🗂️
 
 For the versions available, see the [tags on this repository](https://github.com/idealista/airflow-role/tags).
 
-Additionaly you can see what change in each version in the [CHANGELOG.md](CHANGELOG.md) file.
+Additionally you can see what change in each version in the [CHANGELOG.md](CHANGELOG.md) file.
```
Comment (dortegau) [inline README.md:126]: Maybe it's a good opportunity to [create an issue](https://github.com/idealista/cookiecutter-ansible-role/issues) to fix this typo :)

**Review by dortegau — APPROVED** [review]

File: defaults/main/main.yml [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+  # - { name: SQLAlchemy, version: 1.3.23 } # v1.4 Breaks airflow installation
+#   Version is not mandatory
+#   - {name: pyasn1, version: 0.4.4}
+
+# Owner
+airflow_user: airflow
+airflow_group: airflow
+
+# Admin user
+airflow_admin_users:
```
Comment (xtianae7) [inline defaults/main/main.yml:140]: We might want to leave this empty (add it on molecule variables for testing).
Doing it this way, there will never be a default admin user on playbooks. Users will have to define them manually, removing the risk of having by accident a default admin/admin user/pass :)

Either way, LGTM!

**Review by xtianae7 — APPROVED** [review]

File: defaults/main/main.yml (hunk 3 above)
Comment (ultraheroe) [reply defaults/main/main.yml:140]: Done! and warn about this admin values in README.md

File: .gitignore (hunk 1 above)
Comment (ultraheroe) [reply .gitignore]: Done!

File: README.md (hunk 2 above)
Comment (ultraheroe) [reply README.md:127]: Done!

**Pull request merged**